.github/ISSUE_TEMPLATE/term-request.md

```diff
@@ -1,25 +1,88 @@
----
-name: Term request
-about: Request a new term
-title: ''
-labels: NewTerm
-assignees: ''
-
----
-
-**New term details**
-For us to assess a new term request we require the following details:
-```
-Term name - 
-Structured comment name - [a less than 20char no spaces version of the name]
-Definition - [a clear and concise description of the term including ]
-Expected value - [e.g. text or EFO and/or OBI etc...]
-Value syntax - [e.g. {float} {unit}|{termLabel} {[termID]}|{text}|{timestamp} etc...]
-Example - [provide and example value]
-Preferred unit - [if appropriate]
-Extension(s) - [list any extensions that should include the new term]
-Relationship to other MIXS terms - [list any terms that are dependant on this term or vice-versa, e.g. "pH method" is required when "pH" is given]
-```
-
-**Additional context**
-Add any other context about the new term here.
+name: New Term Request
+description: "For us to assess a new term request we require the following details:"
+title: "New term proposal: "
+labels: [NewTerm]
+body:
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: >
+        Please make sure you check all these items before submitting your feature request.
+      options:
+        - label: I have reviewed the existing MIxS checklists and extensions that the term does not exist.
+          required: true
+  - type: input
+    attributes:
+      label: Name
+      description: >
+        The name of the term. Previously known as 'Term name'
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Title
+      description: >
+        A version of the name with less than 20 characters and no spaces. Previously known as 'Structured comment name'.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Description
+      description: >
+        A clear and concise description of the term. Previously known as 'Definition'.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Short description of pattern or range.
+      description: >
+        The expected type of information the term will hold. Previously known as 'Expected value'.
+      placeholder: >
+        "e.g., text or EFO and/or OBI etc...".
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Pattern or range syntax
+      description: >
+        The pattern of the value that the term will be checked against. Previously known as 'Value syntax'.
+      placeholder: >
+        "e.g. {float} {unit}|{termLabel} {[termID]}|{text}|{timestamp} etc...".
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Example
+      description: >
+        An example value for the term/slot
```

Comment:
should we extend this description to include "the example value must follow the pattern/syntax described above."